projects/nae-example-app/src/app/doc/public-resolver/public-resolver.component.ts

```diff
@@ -25,11 +25,11 @@ export class PublicResolverComponent implements OnInit, OnDestroy {
 
     ngOnInit(): void {
         this._userSub = this._userService.user$.subscribe(user => {
-            if (user.id !== '') {
+            if (!!user && user.id !== '') {
                 this._router.navigate([this.redirectService.parseRedirectPath(this._publicResolver.url)],
                     { queryParams: this.redirectService.queryParams });
                 this._publicResolver.url = undefined;
-            } else if (user.id === '') {
+            } else if (!user || user.id === '') {
```

Comment:
can't this be just an `else` block? Is there some value of `user` where neither the first, nor the second condition evaluates to a truthy value?

Reply:
resolved